superset-frontend/packages/superset-ui-core/src/query/types/Query.ts

```diff
@@ -107,9 +106,6 @@ export interface QueryObject
   /** SIMPLE where filters */
   filters?: QueryObjectFilterClause[];
 
-  /** Time column for SQL, time-grain for Druid (deprecated) */
```

Comment:
I _believe_ this comment is wrong. The time column for SQL is `granularity_sqla` whereas the time column for Druid NoSQL was `granularity`.